src/SFA.DAS.ApplyService.InternalApi/Controllers/GatewayOrganisationChecksController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class GatewayOrganisationChecksController : Controller
+    {
+        private readonly IInternalQnaApiClient _qnaApiClient;
+
+        /// <summary>
+        /// Returns trading name for an application
```

Comment:
Should this comment be on the GetTradingName action?

Reply:
Hah, when I wrote that, there was onl;y one call, and I forgot to update it.  Now done

Reply:
Should the comment exist at all? 😁